spec/controllers/projects/results_controller_spec.rb

```diff
@@ -32,134 +31,23 @@
       }
     end
 
-    subject { post :create, params.merge(project_id: project.id) }
+    subject { post :create, result_params.merge(project_id: project.id) }
 
-    it 'succeeds' do
-      expect(subject).to have_http_status :success
-    end
-
-    it 'creates two results' do
-      expect { subject }.to change { Result.count }.by(2)
+    it { expect(subject).to have_http_status :success }
 
-      expect(Result.all.map(&:tag).uniq).to eq ['pull-request-1']
+    it 'creates the results' do
+      expect { subject }.to change { project.results.count }.by(2)
+      result = project.results.first
+      expect(result.tag).to eq 'pull-request-1'
     end
 
     it 'create a result even no queries are given' do
-      params['result_data'].first.delete 'queries'
+      result_params['result_data'].first.delete 'queries'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.